analyzer/quality-analyzer/src/test/java/eu/fasten/analyzer/qualityanalyzer/MetadataUtilsTest.java

```diff
@@ -0,0 +1,48 @@
+package eu.fasten.analyzer.qualityanalyzer;
+
+import org.junit.jupiter.api.Test;
+
+import static eu.fasten.analyzer.qualityanalyzer.MetadataUtils.normalizeCallableName;
+import static org.junit.jupiter.api.Assertions.*;
+
+class MetadataUtilsTest {
+
+    @Test
+    void normalizeCallableNameJavaTest() {
+        assertEquals("",
+                normalizeCallableName(""));
+
+        assertEquals("NDC.clear",
+                normalizeCallableName("NDC::clear"));
+
+        assertEquals("SMTPAppender.%3Cinit%3E",
+                normalizeCallableName("SMTPAppender::SMTPAppender"));
+
+        assertEquals("LoggingReceiver$Slurper.run",
+                normalizeCallableName("LoggingReceiver::Slurper::run"));
+
+        assertEquals("LoggingReceiver$Slurper.%3Cinit%3E",
+                normalizeCallableName("LoggingReceiver::Slurper::Slurper"));
+
+        assertEquals("PatternParser$NamedPatternConverter.%3Cinit%3E",
+                normalizeCallableName("PatternParser::NamedPatternConverter::NamedPatternConverter"));
+    }
+
+    @Test
+    void normalizeCallableNamePythonTest() {
+        assertEquals("get_filing_list",

```

Comment:
I do not see a difference between expected output and generated output by `normalizeCallableName`. I guess `normalizeCallableName` only works for Java callables?

Reply:
That's true, I only added these Python and C cases to make sure that they are not somehow broken. Also to add more different examples later if we find them (don't think so).